src/ComboboxInput/ComboboxInput.js

```diff
@@ -62,7 +62,8 @@ ComboboxInput.propTypes = {
 };
 
 ComboboxInput.propDescriptions = {
-    menu: 'An object containing a `Menu` component.'
+    menu: 'An object containing a `Menu` component.',
+    popoverProps: 'Additional props to be spread to the [Popover](/popover) component.'
```

Comment:
This actually already exists in the `defaults.js` file albeit with "Popover" marked in backticks rather than a link.  I think making it a link is a good idea, but then it should be edited in `defaults.js` so all components get the same change.